cfgov/cfgov/settings/base.py

```diff
@@ -590,12 +590,7 @@
     "PATH_MATCHES_FOR_QUALTRICS": [],
     # Whether robots.txt should block all robots, except for Search.gov.
     "ROBOTS_TXT_SEARCH_GOV_ONLY": [("environment is", "beta")],
-    # RAFT Testing
-    "RAFT_TESTING": [
-        ("environment is", "beta"),
-        ("after date", "2023-01-22"),
-        ("before date", "2023-01-31"),
-    ],
+    # Whether to render auto-generated page translation links.
```

Comment:
This may have been introduced due to merge messiness?

```suggestion
```

Reply:
Yes, there were some conflicts. I just resolved them, I think!

It seemed like removing that TRANSLATION flag altogether was the desired resolution, correct?

Reply:
Yes, correct, but also this comment should be removed as well, thank you!